Revise Swift entries (#929)
Codecademy/docs: content/swift/concepts/arrays/arrays.md

  - 'paths/build-ios-apps-with-swiftui'
---

> Arrays are ordered collections of values. In other words, an array is an ordered list of values, all of the same type, contained in the same structure. Swift, being a very type safe language, will throw compiler errors if you attempt to store values of different types, say attempting to store a `String` in an `Int` array.

Reviewer: ```suggestion
An **array** is an ordered list of values, all of the same type, contained in the same structure. Swift, being a very type safe language, will throw compiler errors if you attempt to store values of different types, such as attempting to store a `String` in an `Int` array.
```
